managers/memory/missing_syscalls.c

```diff
@@ -284,3 +284,23 @@ SYSCALL_DEFINE1(fsync, unsigned int, fd)
 {
 	BUG();
 }
+
+SYSCALL_DEFINE4(mq_send, char*, name, unsigned long, name_size, unsigned long, msg_size, const char*, msg)
+{
+        BUG();
+}
+
+SYSCALL_DEFINE4(mq_receive, char*, name, unsigned long, name_size, unsigned long*, msg_size, char*, msg)
+{
+        BUG();
+}
+SYSCALL_DEFINE2(mq_close, char*, name, unsigned long, name_size)
+{
+        BUG();
+}
+
+SYSCALL_DEFINE3(mq_open, char*, name, unsigned long, name_size, unsigned long, msg_size)
+{
+	BUG();
+}
+
```

Comment:
These are not implemented??

Reply:
as you don't have sys calls in mComponents, this one is set just to compile successfully in mComponents. Just a hack in LegoOS.